### ISSUE TITLE
Get all tests of Suite

### ISSUE THREAD

**Issue by alex-tiurin** [linkedin/dex-test-parser#42]
Hi. Could you please help me to get the list of all test classes inside SuiteClasses annotation? I didn't find the way to do it.
```kotlin
@RunWith(Suite::class)
@Suite.SuiteClasses(
        TestClassA::class,
        TestClassB::class,
        TestClassC::class
)
class AllSmokeSuite
```
For example, in case above I would like to get TestClassA, TestClassB and TestClassC. After that I want to filter my tests and run only test from these classes.

So, the main purpose is to run the tests of AllSmokeSuite.

**Comment by kkoser** [comment]
Hi! We don't currently support filtering by test suites via the CLI (want to keep it as simple as possible). You could do this with the Java API  manually if you wanted to use it - the class annotations will all be part of the `TestMethod#annotations` field, and you can simply filter your tests out by looking through these annotations for one with the desired. Looks like we were missing support for arrays of values in annotation parsing, I'm putting a small PR up for that now that you'll need

**Comment by kkoser** [comment]
I've merged #43 which adds support for array types in annotation parsing, so you should be able to use that to parse out this annotation, get the values, and then filter teests according to those classes.

**Comment by alex-tiurin** [comment]
Thanks, it works really good! You've saved my day!
